packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -67,7 +67,8 @@ export class CommandQuickOpenItem extends QuickOpenItem {
     }
 
     getKeybinding(): Keybinding | undefined {
-        return this.keybindings.getKeybindingForCommand(this.command.id);
+        const bindings = this.keybindings.getKeybindingsForCommand(this.command.id);
+        return bindings ? bindings[0] : undefined;
```

Comment:
It think we should introduce `keybindings.getPrimaryKeybinding(commandid)`

Reply:
Good idea, and return the first one I guess?

Reply:
for now, yes